### ISSUE TITLE
Do not work on GNOME Shell 3.28.2 Archlinux

### ISSUE THREAD

**Issue by PowaBanga** [linux-man/gnome-shell-extension-printers#7]
Hi,
This really great extention do not work on archlinux, gnome-shell 3.28.2... :(

How can i help to fix that ?

**Comment by linux-man** [comment]
I just tested on Ubuntu 18.04 (Gnome 3.28.1) and it's working. No Arch to test right now.
Please open Looking Glass (ALT+F2 + lg) and see if there's something on Extensions-Printers-Show Errors.
You also can open a terminal, run 'journalctl /usr/bin/gnome-session -f -o cat', restart Gnome (ALT+F2 + r) and paste the output.

**Comment by linux-man** [comment]
I installed Manjaro Gnome Edition 17.1.11 with Gnome-shell 3.28.2 to test Printers.
It worked. No errors. Gnome Control Center and system-config-printers open. Jobs are listed and can be cancelled.
The only thing I found is that "Cancel all jobs" fail because CUPS cancel command gives "cancel-job failed: Forbidden".
Maybe we're dealing with security policies?
I searched a little and it's possible that "boxed instance proxy GIName:GLib.Error" message can be also related to system security. Unfortunately that message is too generic.
